frontend/src/components/Collection.tsx

```diff
@@ -0,0 +1,58 @@
+// External Dependencies
+import { useNavigate } from "react-router-dom";
+
+// Internal Dependencies
+import { ProductsProps } from "../pages/HomePage";
+
+interface CollectionProps {
+  order: number;
+  collections: Array<{
+    image: string;
+    title: string;
+    content: string;
+    urlTitle: string;
+    products: Array<ProductsProps>;
+  }>;
+}
+
+const Collection = ({ order, collections }: CollectionProps) => {
+  const navigate = useNavigate();
+
+  const COLLECTION = collections[order];
```

Comment:
```suggestion
  const collection = collections[order];
```

For variables, it's not recommended to use uppercase letters.